server/helpers/dropSensitiveEventData.js

```diff
@@ -1,3 +1,5 @@
+// source: https://stackoverflow.com/questions/46946380/fetch-api-request-timeout
```

Comment:
Isn't this comment supposed to be placed in `server/helpers/fetchWithTimeout.js`?